### ISSUE TITLE
New command `-build ship`

### ISSUE THREAD

**Issue by tehhowch** [MCOfficer/EndlessSky-Discord-Bot#24]
Totally random at the start, just adds random outfits to the specified ship until its space is full

Next improvement would be to interactively build the ship by giving three choices (numbered) and taking the user's choice. The choice-giving could be incremental (user has to pick one without knowing what the next will be), or all at once.

Final improvement could be storing these builds for recall, e.g. -listbuilds user

**Comment by MCOfficer** [comment]
probably more like an "extra feature" we can go on when there's nothing left to do, but i like the idea :D

**Comment by tehhowch** [comment]
Yeah, definitely an add on. Should be a pretty fun addition though, as everyone who uses it will get something crazy. Will definitely stress-test our java skills

**Comment by PlanetLP** [comment]
What about a non-randomized version as well? Basically, you tell it -build ship and then the outfits with decreasing priority (when the ship is full, it will remove outfits from the bottom of the list until there is enough space), and just ignoring weapons over the maximum number possible, As well as something like -build randomized to get something completely random and so on?
